1/tom-stuart-captions.srt

```diff
@@ -5711,7 +5711,7 @@ to stay in the same order
 1230
 01:16:11,800 --> 01:16:13,740
 in the array, we
-can't use Rubys' sort by.
+can't use Ruby's sort_by.
```

Comment:
The struggle I had was between transcribing the exact words spoken aloud (as it was originally) and the underlying concepts.

Method names are the trickiest example of this: Tom says "descendant count" (not "descendant-underscore-count") but is referring to the method `descendant_count`. The same goes for mentions of class names, e.g. should it be "node" or `Node`?

As long as we are consistent, I am happy with either approach.